var/spack/repos/builtin/packages/gaudi/package.py

```diff
@@ -57,40 +85,33 @@ class Gaudi(CMakePackage):
     depends_on("cmake", type="build")
     depends_on("cppgsl")
     depends_on("fmt", when="@33.2:")
-    depends_on("intel-tbb")
+    depends_on("tbb")
```

Comment:
Leave `intel-tbb` dependency unchanged, but probably needs to be addressed in a separate PR when we can verify that gaudi builds fine with the other `tbb` providers, `intel-oneapi-tbb` and `intel-parallel-studio`.
```suggestion
    depends_on("intel-tbb")
```